docs/configuration/configuration-file.mdx

```diff
@@ -276,25 +514,12 @@ resolve:
         envVariable: SECRET_AUTH
 ```
 
-The first match takes precedence when a URL matches multiple patterns. Therefore, only the header from the first match will be used in the request.
-
-
-### features.openapi
-
-The `features.openapi` section configures features (availability and options) and theming (style, fonts, colors) for API reference documentation generated from API definitions.
-
-There are two products: Redoc Community Edition (CE) and Redocly Reference. Reference supports all configuration options from Redoc Community Edition plus a set of premium options.
-
-Find the full list of supported options on the [Reference docs configuration page](../../api-reference-docs/configuration/index.mdx).
-
-### registry
+If the URL matches multiple patterns, the first match takes precedence. Therefore, only the header from the first match will be used in the request.
 
-The `registry` section is reserved for future API registry features.
+## features.openapi
 
-Read more about the [`registry` section](registry.md).
+The `features.openapi` section is where you configure how your published reference docs will look (style, fonts, colors — known as _theming_) and behave (enable/disable search, show/hide API reference docs download option  — known as _functionality_).
 
-## Schema reference
+Three products are impacted by this section: Redoc Community Edition (CE), Redocly Reference and Developer portal (if the API catalog feature is used). Reference supports all configuration options from Redoc Community Edition plus a set of premium options.
 
-<StyledContent>
-<JsonSchema schema={Schema}/>
-</StyledContent>
+Find the full list of supported options on the [Reference docs configuration page](../../api-reference-docs/configuration/index.mdx) (those marked with **Supported in Redoc CE**).
```

Comment:
question: what about those marked options? I don't understand why this is here - is it just a leftover from some copy/paste?

Reply:
It's here because if you go to the page referenced here (https://redocly.com/docs/api-reference-docs/configuration/theming/), you'll see that options that are supported in Redoc are marked in bold as **Supported in Redoc CE**. Did I get that wrong?

Reply:
This tool is agnostic to if someone is using Redoc or our Developer portal, etc... So, any option there can be used. In fact, I don't think Redoc uses this configuration file format yet.

Reply:
Right, that information is primarily intended for Redoc users who want to customize their docs (because they have limited options for doing that). It's not relevant in this case, so I think you can just remove that whole parenthesis.